### PR TITLE
chore: modify package.json and rollup.config

### PR DESCRIPTION
**Pull request by ssi02014**
## Summary

@cure53 👋
I found that `package.json` has the build commands `umd`, `umd:min`, `es`, and `cjs` set for each of them, and it runs the build through them.

The `rollup.config.js` setting allows you to do the build all at once. It's a minor difference, but it also makes the build go faster. 👍

I checked and the build results before the modification and after the modification were the same.

However, if you decide that this is not for you, you can reject it !! 
Thanks!!

### as-is
스크린샷 2023-11-01 오전 1 54 55

### to-be
스크린샷 2023-11-01 오전 1 55 26

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -3,22 +3,47 @@ const babel = require('@rollup/plugin-babel').babel;
...
+      ...commonOutputConfig,
+      file: pkg.module,
+      format: 'es',
+    },
+    {
+      ...commonOutputConfig,
+      file: pkg.main,
+      format: 'cjs',
+    },
+  ],
```
Comment (ssi02014) [inline rollup.config.js:46]: By setting output to an array, you can set each format and file path.

Also, we can set the plugin individually, so we don't need logic like checking `isProd` to add a `terser`.

File: rollup.config.js [hunk 2]
```diff
@@ -3,22 +3,48 @@ const babel = require('@rollup/plugin-babel').babel;
...
```
Comment (ssi02014) [inline rollup.config.js:47]: By setting output to an array, you can set each format and file path !

File: package.json [hunk 3]
```diff
@@ -7,11 +7,11 @@
...
-    "build": "cross-env run-p build:umd build:umd:min build:es build:cjs",
-    "build:umd": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f umd -o dist/purify.js",
-    "build:umd:min": "cross-env NODE_ENV=production BABEL_ENV=rollup rollup -c -f umd -o dist/purify.min.js",
-    "build:es": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f es -o dist/purify.es.js",
-    "build:cjs": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f cjs -o dist/purify.cjs.js",
+    "build": "cross-env NODE_ENV=production rollup -c",
+    "build:umd": "rollup -c -f umd -o dist/purify.js",
+    "build:umd:min": "rollup -c -f umd -o dist/purify.min.js -p terser",
+    "build:es": "rollup -c -f es -o dist/purify.es.js",
+    "build:cjs": "rollup -c -f cjs -o dist/purify.cjs.js",
```
Comment (ssi02014) [inline package.json:14]: I kept the individual build commands and only changed the internals.
Minimized `env` settings that add complexity.

File: package.json [hunk 4]
```diff
@@ -7,11 +7,11 @@
...
-    "build": "cross-env run-p build:umd build:umd:min build:es build:cjs",
-    "build:umd": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f umd -o dist/purify.js",
-    "build:umd:min": "cross-env NODE_ENV=production BABEL_ENV=rollup rollup -c -f umd -o dist/purify.min.js",
-    "build:es": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f es -o dist/purify.es.js",
-    "build:cjs": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f cjs -o dist/purify.cjs.js",
+    "build": "rollup -c",
+    "build:umd": "rollup -c -f umd -o dist/purify.js",
+    "build:umd:min": "rollup -c -f umd -o dist/purify.min.js -p terser",
+    "build:es": "rollup -c -f es -o dist/purify.es.js",
+    "build:cjs": "rollup -c -f cjs -o dist/purify.cjs.js",
```
Comment (ssi02014) [inline package.json:14]: I kept the individual build commands and only changed the internals.
Minimized env settings that add complexity.

File: rollup.config.js [hunk 5]
```diff
@@ -3,22 +3,48 @@ const babel = require('@rollup/plugin-babel').babel;
...
```
Comment (ssi02014) [inline rollup.config.js:47]: By setting output to an array, you can set each format and file path !

**Comment by cure53** [comment]
Whoa, awesome, thanks again :smile:

**Comment by ssi02014** [comment]
@cure53 Thanks !!! 🎉

**Pull request merged**